db/migrate/20161130152451_create_admin_admission_process_image_details.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAdminAdmissionProcessImageDetails < ActiveRecord::Migration[5.0]
+  def change
+    create_table :admin_admission_process_image_details do |t|
+      t.references :admission_process_image, foreign_key: true
```

Comment:
Please add an index, read story carefully again. If you read it, you will get the process of adding index.